golang/cosmos/app/app.go

```diff
@@ -825,13 +825,25 @@ func NewAgoricApp(
 	return app
 }
 
+func ensureBlockMaxBytes(app *GaiaApp, ctx sdk.Context) {
```

Comment:
Add godoc and rename to make the action more clear.

```suggestion
// capBlockMaxBytes examines the consensus parameters and sets Block.MaxBytes to
// gaiapparams.MaxBlockSizeBytes if it exceeded this value.
func capBlockMaxBytes(app *GaiaApp, ctx sdk.Context) {
```